apps/kol-amelamdim/components/navbar/Navbar.tsx

```diff
@@ -26,23 +26,21 @@ const StyledNavbar = styled(AppBar)`
 export const Navbar = () => {
   const router = useRouter();
 
-  const { isAuthenticated, setAuthenticated } = useContext(AuthContext);
+  const { isAuthenticated, setAuthenticated, checkAuthentication } =
+    useContext(AuthContext);
   const { setAlertMessage, setAlertType } = useContext(AlertContext);
 
-  const checkAuthentication = async () => {
-    try {
-      const { data } = await axios.get('/api/me');
-      if (data.success) {
-        setAuthenticated(true);
-      }
-    } catch (error) {
-      setAlertType('warning');
-      setAlertMessage(error.response.data.message.heb);
-    }
-  };
-
   useEffect(() => {
-    checkAuthentication();
+    checkAuthentication()
+      .then((data) => {
+        if (data.success) {
+          setAuthenticated(true);
+        }
+      })
+      .catch((error) => {
+        setAlertType('warning');
+        setAlertMessage(error.response.data.message.heb);
```

Comment:
for now lets check if the `error.response` is defined then use our messages, otherwise just console.error the error